windows/OrientationWindows/OrientationWindows.cpp

```diff
@@ -0,0 +1,211 @@
+#include "pch.h"
+#include "OrientationWindows.h"
+
+using namespace winrt::Windows::Graphics::Display;
+using namespace winrt::Windows::UI::ViewManagement;
+using namespace winrt::Windows::Devices::Sensors;
+
+namespace OrientationWindows {
+
+    void OrientationLockerModule::Initialize(React::ReactContext const& reactContext) noexcept {
+        m_context = reactContext;
+        m_deviceOrientationSensor = SimpleOrientationSensor::GetDefault();
+        if (m_deviceOrientationSensor != nullptr)
+        {
+            m_deviceOrientationChangedToken = m_deviceOrientationSensor.OrientationChanged({ this, &OrientationLockerModule::OnDeviceOrientationChanged });
+        }
+        else {
+            printf("No Simple Orientation Sensor found on this device. Unable to get device orientation information.");
+        }
+    }
+
+    void OrientationLockerModule::GetOrientation(std::function<void(std::string)> cb) noexcept {
+        DisplayOrientations currentOrientation = m_displayInfo.CurrentOrientation();
```

Comment:
use const wherever possible, and AAA ("almost always auto"):

```suggestion
        const auto currentOrientation = m_displayInfo.CurrentOrientation();
```

Reply:
(same in other places)